stdlib/importlib/machinery.pyi

```diff
@@ -113,10 +114,10 @@ class PathFinder:
 
     @classmethod
     def find_spec(
-        cls, fullname: str, path: Sequence[bytes | str] | None = ..., target: types.ModuleType | None = ...
+        cls, fullname: str, path: Sequence[str] | None = ..., target: types.ModuleType | None = ...
```

Comment:
The path gets passed to `_get_spec`, which ignores any non-str entries in the path: https://github.com/python/cpython/blob/2cfcaf5af602b297fc90086de4d8ac980c7891e2/Lib/importlib/_bootstrap_external.py#L1525